src/pudl/io_managers.py

```diff
@@ -658,6 +658,8 @@ def load_input(self, context: InputContext) -> pd.DataFrame:
         ferc1_settings = context.resources.dataset_settings.ferc1
 
         table_name = self._get_table_name(context)
+        # Remove preceeding asset name metadata
+        table_name = table_name.replace("raw_ferc1_dbf__", "")
```

Comment:
I wonder if we want to write a utility function that gets the base table name - do you think that would be helpful in the future? 

```python
def get_base_table_name(s): # or... strip_metadata(s)?
    return s.split('__', 1)[-1]
```

I guess at least it would get rid of the need for this comment 😅